src/main/java/com/iexec/resultproxy/jwt/Jwt.java

```diff
@@ -16,16 +16,18 @@
 
 package com.iexec.resultproxy.jwt;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.annotation.Version;
 import org.springframework.data.mongodb.core.index.Indexed;
 import org.springframework.data.mongodb.core.mapping.Document;
 
-@Data
+@Getter
 @Document
```

Comment:
```suggestion
@Document
@Getter
```
Minor: switch both annotations so that JPA annotations are not mixed up with Lombok annotations.

Reply:
Good suggestion